static/app/utils/profiling/flamegraphChart.tsx

```diff
@@ -0,0 +1,57 @@
+import {Rect} from 'sentry/utils/profiling/speedscope';
+
+import {makeFormatter} from './units/units';
+
+interface Serie {
+  points: {x: number; y: number}[];
+}
+
+export class FlamegraphChart {
+  configSpace: Rect;
+  formatter: ReturnType<typeof makeFormatter>;
+  series: Serie[];
+  domains: {
+    x: [number, number];
+    y: [number, number];
+  } = {
+    x: [0, 0],
+    y: [0, 0],
+  };
+
+  static Empty = new FlamegraphChart(Rect.Empty(), {unit: 'percent', values: []});
+
+  constructor(configSpace: Rect, measurement: Profiling.Measurement) {
+    this.series = new Array<Serie>();
+
+    this.series[0] = {
+      points: new Array(measurement.values.length),
+    };
+
+    for (let i = 0; i < measurement.values.length; i++) {
+      const m = measurement.values[i];
+
+      // Track and update Y max and min
+      if (m.value > this.domains.y[1]) {
+        this.domains.y[1] = m.value;
+      }
+      if (m.value < this.domains.y[0]) {
+        this.domains.y[0] = m.value;
```

Comment:
We can test this but it might be better to always start the y axis for the CPU chart at 0.

Reply:
Yeah, we'll test all of this. With the initial data I was seeing, all of it was +-1% so it was just one flat line. But yeah, we'll start with 0-100 all the time

Reply:
Just checked, the data is initialized to 0 so unless SDKs report < 0, this wont ever kick in